apps/rig_auth/lib/rig_auth/jwt/utils.ex

```diff
@@ -12,19 +12,27 @@ defmodule RigAuth.Jwt.Utils do
   @type claims :: %{required(String.t()) => String.t()}
 
   @spec valid?(String.t()) :: boolean
-  def valid?(jwt) do
+  def valid?("Bearer " <> jwt) do
     jwt
     |> validate
     |> get_error == nil
   end
 
+  def valid?(jwt) do
+    %{error: "JWT=#{jwt} has incorrect form. Required form is: \"Bearer token\""}
```

Comment:
Probably being picky here, but
- the spec doesn't call this "form"
- a non-bearer type token might not be supported by RIG but that doesn't necessarily mean it's an "incorrect" token
- the variable shouldn't be called `jwt` as it's most likely _not_ a JWT at this point